samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -72,6 +72,38 @@
           // adjust the height of the iframe to accomodate the expansion from the accordion
           $("body").height($("body").outerHeight() + 900);
 
+          checkNav = function() {
+              QuesFormatRadios = ["assessmentSettingsAction\\:assessmentFormat\\:0", "assessmentSettingsAction\\:assessmentFormat\\:1", "assessmentSettingsAction\\:assessmentFormat\\:2"];
+
+              enabled = true;
+              if ($("#assessmentSettingsAction\\:itemNavigation\\:0:checked").val() !== undefined) {
```

Comment:
I believe preferred jquery way to do this is $(selector).is(':checked')

Reply:
Ok, I'll change it.

Reply:
Testing this change made me discover another problem/optimization opportunity: the lines that disabled and toggled the checked state of the 'Mark for Review' checkbox were inside the foreach iterating over the radio buttons. So that part only really worked because there's an odd number of radios. Just had to move those lines outside of the loop.

So thanks again for the extra set of eyes :)

Reply:
odd number of radios. nice!